src/shared/services/configuration/endpoint-definitions.js

```diff
@@ -95,6 +95,8 @@ export default {
   // for reference.
   STANDARD_TEXTS_SEARCH_ENDPOINT: '/signals/v1/private/status-messages/search',
   // Private. Used by the incident management module to retrieve and search standard texts.
+  STANDARD_TEXTS_DETAILS_ENDPOINT: '/signals/v1/private/status-messages/',
+  // Private. Used by the incident management module to retrieve the details of a standard text.
```

Comment:
It is to retrieve data, but also patch, post and delete.

Reply:
This is old, right?

Reply:
added info about the patch and delete